tests/commands/test_groups.py

```diff
@@ -135,6 +141,30 @@ def test_groups_group_command__create_HTTPError(mock_client, capfd):
     assert "Matching groups (3)" in capture.out
 
 
+def test_groups_group_command__funding_sources(mock_client, capfd):
+    mock_client.get_concession_group_linked_funding_sources.return_value = (r for r in GROUP_FUND_RESPONSES)
+
+    args = Namespace(group_command="funding_sources")
+    res = groups(args)
+    capture = capfd.readouterr()
+
+    assert res == RESULT_SUCCESS
+    assert "Matching groups (3)" in capture.out
+    assert "💵 Linked funding sources " in capture.out
```

Comment:
Maybe add the expected number after `Linked funding sources` here, just like for `Matching groups (3)`?

Reply:
Yep, I agree! I also added two leading spaces I was missing in the string, to match the command more closely.